src/libponyrt/lang/socket.c

```diff
@@ -847,6 +847,14 @@ PONY_API struct addrinfo* pony_os_addrinfo(int family, const char* host,
   return os_addrinfo_intern(family, 0, 0, host, service, true);
 }
 
+/*
```

Comment:
This change was done only for reference and should NOT have been committed. I apologize for the inconvenience. Please help me withdraw/cancel this pull request.